packages/react-components/react-popover/library/src/components/PopoverSurface/usePopoverSurfaceStyles.styles.ts

```diff
@@ -53,7 +53,28 @@ const useStyles = makeStyles({
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32138&commitId=5ebf5a2703ef68eb12f322261dd5622ff7d9f587&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|6| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|2| Changed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|16| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|14| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|13| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>